ed-lti.php

```diff
@@ -25,7 +25,12 @@
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-require_once 'classes/class-ed-lti.php';
+namespace EdLTI;
+
+// Include the autoloader so we can dynamically include the rest of the classes.
```

Comment:
This comment should be above the section where we load the autoloader.

Reply:
done